web-common/src/metrics/ErrorEventHandler.ts

```diff
@@ -36,4 +38,29 @@ export class ErrorEventHandler {
       glob,
     ]);
   }
+
+  public fireErrorBoundaryEvent(api: string, status: string, message: string) {
+    return this.metricsService.dispatch("errorBoundaryEvent", [
+      this.commonUserMetrics,
+      this.getScreenNameFromPage(),
+      api,
+      status,
+      message,
+    ]);
+  }
+
+  private getScreenNameFromPage() {
```

Comment:
One option is to leverage the functions in `web-admin/features/navigation/nav-utils.ts`. Particularly for the project page, we should make sure we're capturing all the sub-routes.